bundler.gemspec

```diff
@@ -37,10 +37,12 @@ Gem::Specification.new do |s|
   s.add_development_dependency "ronn",       "~> 0.7.3"
   s.add_development_dependency "rspec",      "~> 3.6"
 
-  s.files = `git ls-files -z`.split("\x0").reject {|f| f.match(%r{^(test|spec|features)/}) }
+  s.files = `git ls-files -z`.split("\x0").select {|f| f.match(%r{^(lib|exe)/}) }
   # we don't check in man pages, but we need to ship them because
   # we use them to generate the long-form help for each command.
   s.files += Dir.glob("man/**/*")
+  # Include the CHANGELOG.md, LICENSE.md, README.md manually
+  s.files += %w[CHANGELOG.md LICENSE.md README.md]
```

Comment:
Thought that the `CODE_OF_CONDUCT.md` and `CONTRIBUTING.md` wouldn't be useful with a rubygems installed `bundler`, and more useful to a developer cloning this repo with the intent to contribute.  That said, the `TROUBLESHOOTING.md` doc mentioned in the `README.md` might be useful, so there is that.

Happy to add or remove things as you see fit, just picked a few the seemed the most relevant to start.